book.rb

```diff
@@ -0,0 +1,8 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
+end
```

Comment:
- [ ] kindly ensure that the `many-to-many` (also `has-many-through`) relationship between `Person` and `Book` using the intermediate class Rentals is implemented 👍 as required 😄 

![Capture](https://github.com/Bupilipili/OOP-school-library/assets/80458199/64fd9ed9-8d58-47e2-a1a4-6cfda75871df)


For more information about that , Kindly check this [link ](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)